src/test/java/unit/BqDatasetReferenced.java

```diff
@@ -309,60 +309,64 @@ void updateIndividualProperties() throws IOException {
     // update just the name
     // `terra resources update bq-dataset --name=$name --new-name=$newName`
     String newName = "updateIndividualProperties_NEW";
-    UFBqDataset updateDataset =
+    UFBqDataset updatedDataset =
         TestCommand.runAndParseCommandExpectSuccess(
             UFBqDataset.class,
             "resource",
             "update",
             "bq-dataset",
             "--name=" + name,
             "--new-name=" + newName);
-    assertEquals(newName, updateDataset.name);
-    assertEquals(description, updateDataset.description);
+    assertEquals(newName, updatedDataset.name);
+    assertEquals(description, updatedDataset.description);
 
     // `terra resources describe --name=$newName`
-    UFBqDataset describeDataset =
+    UFBqDataset describedDataset =
         TestCommand.runAndParseCommandExpectSuccess(
             UFBqDataset.class, "resource", "describe", "--name=" + newName);
-    assertEquals(description, describeDataset.description);
+    assertEquals(description, describedDataset.description);
 
     // update just the description
-    // `terra resources update bq-dataset --name=$newName --description=$newDescription`
+    // `terra resources update bq-dataset --name=$newName --description=$newDescription
+    // --new-cloning=COPY_REFERENCE`
     String newDescription = "updateDescription_NEW";
-    updateDataset =
+    updatedDataset =
         TestCommand.runAndParseCommandExpectSuccess(
             UFBqDataset.class,
             "resource",
             "update",
             "bq-dataset",
             "--name=" + newName,
-            "--description=" + newDescription);
-    assertEquals(newName, updateDataset.name);
-    assertEquals(newDescription, updateDataset.description);
+            "--description=" + newDescription,
+            "--new-cloning=" + CloningInstructionsEnum.DEFINITION);
+    assertEquals(newName, updatedDataset.name);
+    assertEquals(newDescription, updatedDataset.description);
+    assertEquals(newDescription, updatedDataset.description);
```

Comment:
Looks like you have this line twice. Also, you're not asserting anything about the cloning instructions. I believe I fixed the bug that required doing a describe before checking it.